packages/terra-table/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -130,6 +130,12 @@ const propTypes = {
    * Object containing intl APIs
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }),
+
+  /**
+   * @private
+   * The information to be conveyed to screen readers about the highlighted column.
+   */
+  columnHighlightDescription: PropTypes.string,
```

Comment:
Shouldn't the column header cell be able to determine this information without it being passed in?  I would expect that this component could centralize that logic.

Reply:
This prop was introduced to allow consumer specified context to be announced for such columns
This was an a11y requirement: https://github.com/cerner/terra-framework/pull/1985#pullrequestreview-1853859162  
Each highlighted column can have its own specific description.